library/billing.inc

```diff
@@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
           "pid = '$patient_id' AND encounter = '$encounter_id'");
       }
     }
```

Comment:
@teryhill 
Just checking in with you. Weren't most of the above if (GLOBALS['oer_config']['ws_accounting']['enabled']) stuff removed by the sql-ledger removal?
(there are a couple more on a grep, and will then likely remove them with this PR)